core/src/generate_types.rs

```diff
@@ -114,7 +114,7 @@ impl<'a> TypeGenerator<'a> {
             TypeDef::Sequence(seq) => vec![seq.type_param().id()],
             TypeDef::Tuple(tuple) => tuple.fields().iter().map(|f| f.id()).collect(),
             TypeDef::Compact(compact) => vec![compact.type_param().id()],
-            TypeDef::Phantom(phantom) => vec![phantom.type_param().id()],
+            TypeDef::Phantom(_phantom) => vec![/* TODO [now]: this is not yet in the `aj-substrate` branch phantom.type_param().id() */],
```

Comment:
Actually I have remove that here: https://github.com/paritytech/scale-info/pull/96, and `aj-substrate` includes that PR. We need to figure out whether we still have enough information to generate the correct types. We probably should, but it needs testing out on this side.